lib/matplotlib/mathtext.py

```diff
@@ -769,7 +769,18 @@ class UnicodeFonts(TruetypeFonts):
 
     def __init__(self, *args, **kwargs):
         # This must come first so the backend's owner is set correctly
-        if rcParams['mathtext.fallback_to_cm']:
+        fallback_rc = rcParams['mathtext.fallback']
```

Comment:
Could be written more compact as
~~~
        fallback_rc = rcParams['mathtext.fallback']
        if rcParams['mathtext.fallback_to_cm'] is not None:
            fallback_rc = ('cm' if rcParams['mathtext.fallback_to_cm']
                           else None)

        font_class = {'stix': StixFonts,
                      'stixsans': StixSansFonts,
                      'cm': BakomaFonts
                      }.get(fallback_rc)
        self.cm_fallback = font_class(*args, **kwargs) if font_class else None
~~~
and after the deprecation further simplified to
~~~
        font_class = {'stix': StixFonts,
                      'stixsans': StixSansFonts,
                      'cm': BakomaFonts
                      }.get(rcParams['mathtext.fallback'])
        self.cm_fallback = font_class(*args, **kwargs) if font_class else None
~~~